src/ngraph/op/op.tbl

```diff
@@ -0,0 +1,112 @@
+//*****************************************************************************
+// Copyright 2017-2018 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+// This collection contains one entry for each op. If an op is added it must be
+// added to this list.
+//
+// In order to use this list you want to define a macro named exactly NGRAPH_OP_LIST
+// When you are done you should undef the macro
+// As an example if you wanted to make a list of all op names as strings you could do this:
+//
+// #define NGRAPH_OP_LIST(a) #a,
+// std::vector<std::string> op_names{
+// #include "this include file name"
+// };
+// #undef NGRAPH_OP_LIST
+//
+// This sample expands to a list like this:
+// "Abs",
+// "Acos",
+// ...
+//
+// It's that easy. You can use this for fun and profit.
+
+NGRAPH_OP_LIST(Abs)
```

Comment:
Should be `NGRAPH_OP` -- these are items of a list, not lists.

Reply:
done